### ISSUE TITLE
Unify configmap + improve installer (USAGE & secrets & update-ymls)

### ISSUE THREAD

**Issue by shay-berman** [IBM/ubiquity-k8s#112]
**1. Unify configuration :**
Currently there are many parameters that the use should set in multiple yml files.
The goal of this ticket is to simplify the installation experience by consolidate the setting to 2 configmap:
- [ubiquity-configmap] with all the ubiquity deployment parameters  + shared paramaters like SSL_MODE, LogLevel (that shared between: ubiquity, ubiquity-db, ubiquity-k8s-provisioner and ubiquity-k8s-flex)

In addition, for credentials (which also spread on different ymls) we will create the following secrets:
- [scbe-credentials] for SCBE credential needed for deployments : ubiquity, ubiquity-k8s-provisioner and ubiquity-k8s-flex
- [ubiquity-db-credentials] for Ubiquity DB credential needed for deployments : ubiquity, ubiquity-db

**2. Improve installer USAGE**
Align installer to use the same convention -s <STEP> for all the operations.
Below full USAGE description

```
USAGE   $cmd -s <STEP> <FLAGS>
  <STEP>:
    -s update-ymls -c <file>
        Replace the placeholders from -c <file> on the relevant ymls.
        Flag -c <ubiquity-config-file> is mandatory for this step

    -s install [-k <k8s config file>] [-n <namespace>]
        Install all ubiquity component by order (except for ubiquity-db)
        Flag -k <k8s-config-file-path> for ubiquity-k8s-provisioner. By default ~/.kube/conf.
        Flag -n <namespace>. By default its \"ubiquity\" namespace.
    -s create-ubiquity-db [-n <namespace>]
        Creates only the ubiquity-db deployment and waits for its creation.
        Use this option after install-ubiquity finished and after manual restart of the kubelets done on the nodes.
        Flag -n <namespace>. By default its \"ubiquity\" namespace.

    -s create-services [-n <namespace>]
        Use it only if SSL_MODE=verify-full
        This step creates ubiquity namespace, ubiqutiy and ubiquity-db k8s service only.
        If in order to create certificates you need to see the ubiquity and ubiqity-db IPs, then run this step.
        Flag -n <namespace>. By default its \"ubiquity\" namespace.
    -s create-secrets-for-certificates -t <certificates-directory>  [-n <namespace>]
        Use it only if SSL_MODE=verify-full
        Creates secrets and configmap for ubiquity certificates:
            Secrets ubiquity-private-certificate and ubiquity-db-private-certificate.
            Configmap ubiquity-public-certificates.
        Flag -t <certificates-directory> that contains all the expected certificate files.
 -h : Display this usage
```

**3. Formalize the update-ymls STEP**
Provide this new STEP for the customer to simplify the updates of the Configmap \ Secrets \ Storage class and PVC.
So instead of update manual each one of these ymls, the customer will update only ubiquity.config file, and the update-yml STEP will update the relevant keys to the relevant ymls automatically (including encoding to base64 for Secrets values like username and password).

**Comment by shay-berman** [comment]
After feadback we did some changes to the design :
1. To reduce k8s configmap objects, I moved params from ubiquity-deployments-configmap to the ubiquity-configmap.
2. already refactor the scbe-credentials and ubiquity-db-credentials to be secrets objects instead of configmap.

Note : Consider to merge scbe-credentials and ubiquity-db-credentials to a single secret (again to reduce k8s secret objects).
